packages/services/storage/tools/register.cjs

```diff
@@ -1,3 +0,0 @@
-const { register } = require('esbuild-register/dist/node');
-
-register({ extensions: ['.mjs', '.ts'], format: 'cjs' });
```

Comment:
I removed this file for the following reasons
1. removing `esbuild-register`
2. removing `strorage/migrations/package.json`
3. removing `storage/tools/register.cjs`
4. exec `db:migrator` script with `node --experimental-specifier-resolution=node --loader ts-node/esm migrations/migrate.ts` as `ts-node` already installed in package.json

Reply:
the thing is, it has to run in docker and as cli, it it works, I'm fine with the changes.